.eslintrc

```diff
@@ -22,7 +22,8 @@
     "no-underscore-dangle": 0,
     "import/prefer-default-export": 0,
     "import/imports-first": ["error", "absolute-first"],
-    "import/newline-after-import": "error"
+    "import/newline-after-import": "error",
+    "react/state-in-constructor": ["warn", "never"]
```

Comment:
Why you add this rule? It's not bad to use constructors.

Reply:
Because eslint underline my code.